src/main/java/org/zalando/nakadi/service/timeline/TimelineService.java

```diff
@@ -154,6 +156,24 @@ public void createTimeline(final String eventTypeName, final String storageId)
         }
     }
 
+    public void repartitionET(final EventType eventType, final int partitions) throws
```

Comment:
I think it makes sense to have more meaningful name, like `repartitionEventType`

Reply:
or `repartitionTimeline`